### PR TITLE
Fix error in Kibana dashboard record tables

### LINKED ISSUE #14127: Error in Kibana dashboard record tables
**Describe the issue:**
Kibana dashboard record tables for Auth and Session dashboards displaying error under heavy load. The error is as follows
Screenshot 2022-07-06 at 14 42 27

**How to reproduce:**
Produce a heavy load (Around 18000 record in 10 minutes time span) of Authentication operations.

**Expected behavior:**
Records should be shown in the table.

**Environment information** 
 - Product Version: IS 6.0.0 Alpha
 - OS: Ubuntu 22.04 (AWS)
 - Database: MySQL (AWS RDS)
 - Userstore: JDBC

### PR DESCRIPTION
**Pull request by Avarjana**
Replaced the Lens table visualisation with [Kibana enhanced table plugin](https://github.com/fbaligand/kibana-enhanced-table) to solve the error caused by aggregation limit reached on Kibana dashboards.

Fixes https://github.com/wso2/product-is/issues/14127

### CODE REVIEW

**Review by IsankaSR — APPROVED** [review]

**Comment by IsankaSR** [comment]
Have you added the instructions in relavant docs to config **Kibana enhanced table plugin**?

**Comment by Avarjana** [comment]
> Have you added the instructions in relavant docs to config **Kibana enhanced table plugin**?

Yes. https://github.com/wso2/docs-is/pull/3137

File: kibana/saved-objects/kibana-8-x-auth-and-session.ndjson [hunk 1]
```diff
@@ -1,34 +1,34 @@
...
 {"excludedObjects":[],"excludedObjectsCount":0,"exportedCount":33,"missingRefCount":0,"missingReferences":[]}
```
Comment (lashinijay) [inline kibana/saved-objects/kibana-8-x-auth-and-session.ndjson:34]: Shall we add a new line here?

Comment (Avarjana) [reply kibana/saved-objects/kibana-8-x-auth-and-session.ndjson:34]: Addressed in https://github.com/wso2-extensions/identity-elk-integration/pull/14

**Pull request merged**